reactivesocket-cpp/src/Frame.h

```diff
@@ -8,15 +8,19 @@
 
 /// Needed for inline d'tors of frames.
 #include <folly/io/IOBuf.h>
+#include <folly/io/IOBufQueue.h>
+#include <folly/Memory.h>
+#include <folly/Optional.h>
+#include <folly/io/Cursor.h>
```

Comment:
you don't have to pull in this header if you are just declaring methods with the folly::Cursor symbol.
it is declared on line 22.

you will need to include the header file in the cpp file where you will call methods off of folly::Cursor.